src/configure.rs

```diff
@@ -61,17 +64,12 @@ pub fn update_configuration(name: &str, value: &str) {
 
 pub fn edit_configuration() {
     let config_path = get_config_path();
-    let editor_cmd = get_editor();
+    let editor_cmd = shell_words::split(&get_editor()).unwrap();
```

Comment:
`unwrap()` always makes me just a little nervous. Under what conditions can this fail? Is it a realistic failure mode for us? At the very least, it would be good to document when the split might fail (in this case, only when unmatched quotes are found in the result of `get_editor()`)

Reply:
The self-documenting option for this is `.expect("Unmatched quotes found in editor")`

Reply:
Yes, I should have use `expect` in the first place here.